src/2025h1/restrictions.md

```diff
@@ -0,0 +1,81 @@
+# Implement restrictions, prepare for stabilization
+
+| Metadata |          |
+| -------- | -------- |
+| Owner(s) | @jhpratt |
+| Teams    | [lang]   |
+| Status   | Proposed |
+
+[rfc]: https://rust-lang.github.io/rfcs/3323-restrictions.html
+[pr]: https://github.com/rust-lang/rust/pull/106074
+
+## Summary
+
+[RFC 3323][rfc] will be implemented and feature-complete, with all syntax questions resolved. The
+features will be prepared for stabilization.
+
+## Motivation
+
+The [RFC for restrictions][rfc] was accepted over two years ago, but the [pull request implementing
+it][pr] has been stalled for a long time for a variety of reasons. Implementing the feature will
+permit testing, feedback, and stabilization.
+
+### The status quo
+
+Sealed traits are a common pattern in Rust, but are not currently supported by the language itself.
+Instead, they are implemented using a combination of visibility modifiers and nested modules. Fields
+with restricted mutability are currently only possible with getters and setters, setting aside
+(ab)using `Deref` implementations.
+
+More details are available in [the RFC][rfc].
+
+### The next 6 months
+
+The [accepted restrictions RFC][rfc] represents the end goal of this project goal. All unresolved
+questions should be discussed and resolved, with the two features (`impl_restrictions` and
+`mut_restrictions`) being ready for stabilization. Future possibilities are likely considered at a
+high level, but are not the focus of this project goal.
+
+## Ownership and team asks
+
+**Owner:** @jhpratt
+
+| Subgoal                                 | Owner(s) or team(s)  | Notes                                      |
+| --------------------------------------- | -------------------- | ------------------------------------------ |
+| Discussion and moral support            | ![Team][] [lang]     |                                            |
+| Stabilize restrictions                  |                      |                                            |
+| ↳ Implementation                        | @jhpratt             | [old PR][pr] is plausibly workable         |
+| ↳ Code reviews                          | ![Team][] [compiler] |                                            |
+| ↳ Design decisions                      | ![Team][] [lang]     | for unresolved questions, including syntax |
+| ↳ Author stabilization report           | @jhpratt             |                                            |
+| ↳ Stabilization decision                | ![Team][] [lang]     |                                            |
```

Comment:
Do you expect to reach stabilization in the next 6 months?

Reply:
I think it's a reasonable expectation to set. The implementation will absolutely be done, and there's only a couple unresolved questions. It may not be stabilized but should at least be _ready_ for it.